README.md

```diff
@@ -27,19 +27,15 @@ Building
 
 To build relx and generate a standalone escript executable:
 
-    $ make
+    $ ./rebar3 escriptize
```

Comment:
No, it doesn't run test. Remember we removed default running of tests from `make` long ago as well :P